content/middlewares.md

```diff
@@ -122,6 +122,8 @@ export class AppModule {
 
 > info **Hint** The `configure()` method can be made asynchronous using `async/await` (e.g., you can `await` completion of an asynchronous operation inside the `configure()` method body).
 
+> warning **Warning** When using the `express` adapter, the NestJS app will register `json` and `urlencoded` from the package `body-parser` by default. This means if you want to customize that middleware per route, you need to turn off the global middleware by setting the `bodyParser` flag to `false` when creating the application with `NestFactory.create()`.
```

Comment:
```suggestion
> warning **Warning** When using the `express` adapter, the NestJS app will register `json` and `urlencoded` from the package `body-parser` by default. This means if you want to customize that middleware via the `MiddlewareConsumer`, you need to turn off the global middleware by setting the `bodyParser` flag to `false` when creating the application with `NestFactory.create()`.
```

You can still do per route middleware via `app.use()` in the same syntax as you would with express. `app.use('/some-path', middleware)`. Being explicit that this is only for the `MiddlewareConsumer` I think is a better option here